modules/__tests__/enhancer-test.js

```diff
@@ -155,14 +155,19 @@ describe('Enhancer', () => {
     });
   });
 
-  it('manually populates static properties for IE <10', () => {
-    class Composed extends Component { }
+  it('manually populates all static properties for IE <10', () => {
+    class Composed extends Component {
+      static staticMethod() {
```

Comment:
Hah, I literally just pushed a fix for this as you typed this - I only just realized this was failing. It'd be nice if a fail triggered an e-mail alert of some sort :)

Yeah I am not a fan of this lint rule - `fn()` > `fn ()` as far as I'm concerned!